DEVELOPMENT.md

```diff
@@ -19,6 +19,18 @@ sh preflight.sh
 mise trust --yes
 mise trust --yes tests
 mise install
+```
```

Comment:
This reads to me now like only the first code block is needed for the "Local development quickstart" section (when both code blocks are still necessary for getting started).

Might be clearer to hoist the new info into the "Before you quickstart" callout or to make "Local development quickstart" a proper heading.

Reply:
Ah yep, makes sense!